src/modules/project/eventHandlers/onDateMiseEnServiceTransmise.ts

```diff
@@ -79,85 +74,102 @@ export const makeOnDateMiseEnServiceTransmise =
             délaiCDC2022appliqué,
             completionDueOn,
           }) => {
-            const projectAppelOffre = getProjectAppelOffre({
+            const délaiCDCApplicable = getProjectAppelOffre({
               appelOffreId,
               periodeId,
               familleId,
-            });
+            })?.periode.cahiersDesChargesModifiésDisponibles.find((CDC) =>
+              'alternatif' in cahierDesCharges
+                ? CDC.type === 'modifié' &&
+                  CDC.paruLe === '30/08/2022' &&
+                  CDC.alternatif === cahierDesCharges.alternatif
+                : CDC.type === 'modifié' && CDC.paruLe === '30/08/2022',
+            )?.délaiApplicable;
```

Comment:
À mon avis t'es pas obligé de vérifier l'alternatif, car meme si il est undefined, il devrait l'etre également dans le cahierDesCharges. Je pense que tu peux simplement faire :


```suggestion
const délaiCDCApplicable = getProjectAppelOffre({
              appelOffreId,
              periodeId,
              familleId,
            })?.periode.cahiersDesChargesModifiésDisponibles.find((CDC) => CDC.type === 'modifié' && CDC.paruLe === '30/08/2022' && CDC.alternatif === cahierDesCharges.alternatif)?.délaiApplicable;
```